Common/src/main/java/com/blamejared/crafttweaker/impl/script/RecipeManagerScriptLoader.java

```diff
@@ -29,7 +30,9 @@ public static void loadScriptsFromManager(RecipeManager manager) {
         accessRecipeManager.getRecipes().replaceAll((k, v) -> new HashMap<>(accessRecipeManager.getRecipes().get(k)));
         CraftTweakerAPI.setRecipeManager(manager);
         
-        final ScriptLoadingOptions scriptLoadingOptions = new ScriptLoadingOptions().setSource(ScriptLoadingOptions.CLIENT_RECIPES_UPDATED_SCRIPT_SOURCE)
+        final ScriptLoadingOptions scriptLoadingOptions = new ScriptLoadingOptions()
+                .setSource(ScriptLoadingOptions.CLIENT_RECIPES_UPDATED_SCRIPT_SOURCE)
+                .setRunConfiguration(CraftTweakerDefaultScriptRunConfiguration.DEFAULT_CONFIGURATION)
```

Comment:
Do we want this in a `.setDefaultRunConfiguration()` call?
Then we have more freedom moving the other class later, and most modders would probably only ever use that instead of the other one anyways 🤔